spec/registry.dtd

```diff
@@ -1,6 +1,6 @@
-<!ELEMENT registry (function*|pattern*)>
+<!ELEMENT registry (function,pattern)*>
```

Comment:
Thanks for noticing this. I filed #434 to discuss and fix this independently of this PR, so that you can focus it on `registry.xml`.